handbook/engineering/cloud/index.md

```diff
@@ -6,6 +6,26 @@ The cloud team owns all work that is necessary to build, secure, scale, and oper
 
 1. Any user or organization can use Sourcegraph Cloud with both the private and public code that they care about in a secure way.
 
+## Milestones
+
+1. Any user can add public code
+2. Organizations can use Sourcegraph with up to 50 repos or 100k LoC of private repositories for free
+    - We need to build a scalable and reliable service to count lines of code and integrate that with the rest of our systems (to enforce limits). This would align with how we’re thinking about billing by LoC, not number of repositories.
+    - No plaintext tokens or secrets anywhere (including gitserver .git/config remotes).
+    - All private repository contents need to be encrypted at rest.
```

Comment:
All this data is stored in Persistent Volumes which are packed by GCP Persistent Disks which are default encrypted at rest by GCP.  I think the term `encrypted-at-rest` might not be specific enough.  What about:
```suggestion
All private repository content are only decryptable by the Sourcegraph instance
```